build.sh

```diff
@@ -30,7 +30,7 @@ pack() {
   echo -e "Pack ...... "
   echo -e "===========================================================\n"
 
-  dotnet pack --no-build WilsonUnix.sln
+  dotnet pack --no-build WilsonUnix.sln -c Debug
```

Comment:
The script builds Debug assets above, it also expects Debug NuGets below. The default configuration for `dotnet pack` is Release though, so this always failed.